docs/installation.rst

```diff
@@ -28,6 +28,12 @@ The easiest way to install the latest stable version of Airflow is with ``pip``:
 .. code-block:: bash
 
     pip install apache-airflow
+    
+If the `airflow` command is not being recognized, then install Airflow globally:
```

Comment:
Installing airflow globally is very dangerous because Airflow requires a large number of dependencies that may not be compatible with other applications.
```
11:50 $ python
Python 3.6.8 (v3.6.8:3c6b436a57, Dec 24 2018, 02:04:31)
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import airflow.bin
>>> airflow.bin.__file__
'/Users/kamilbregula/devel/CLIENT-airflow/airflow/airflow/bin/__init__.py'
>>>
KeyboardInterrupt
>>>
KeyboardInterrupt
>>> ^D
11:53 $ export PATH=/Users/kamilbregula/devel/CLIENT-airflow/airflow/airflow/bin:$PATH
```

Reply:
Yes. Agree with @mik-laj . I think a better (and recommended) solution is to add your `${HOME}/.local/bin` to PATH in .profile. I think you hit this bug of WSL:  https://github.com/microsoft/WSL/issues/2148 - where the PATH does not include the .local/bin folder. Would it be possible @JeremyBernier that you double check it and update it if you see that it works. We have no WSL/Windows handy ....